raiden/api/python.py

```diff
@@ -1715,7 +1715,7 @@ def create_light_client_payment(
 
             # Create the light_client_payment
             is_lc_initiator = 1
-            payment = LightClientPayment(creator_address, partner_address,
+            payment = LightClientPayment(partner_address,
```

Comment:
Why the `creator_address `param was removed?

Reply:
Because in the database it was directly mapped to the LC address and we are removing that from the table so it's not needed anymore.